libvirt/tests/src/migration/async_job/abort_before_qemu_layer_mig_start.py

```
@@ -0,0 +1,97 @@
+        extra = params.get("virsh_migrate_extra")
+        err_msg = params.get("err_msg")
+        abort_mig = params.get("abort_mig")
+
+        mig_t = threading.Thread(target=run_migration, args=(vm_name, dest_uri, option, extra))
+        mig_t.start()
+        time.sleep(3)
+
+        if abort_mig == "domjobabort":
+            virsh.domjobabort(vm_name, debug=True)
+        elif abort_mig == "ctrl_c":
+            ret = process.run("pgrep -f 'virsh migrate'", shell=True, verbose=True).stdout_text.strip()
```

Ruff (0.14.0)

Function call with `shell=True` parameter identified, security issue

(S604)